gaussian_blur_size: 7
gaussian_blur_sigma: 2
canny_th_low: 70.5
canny_th_high: 199.5
min_line_length: 42
max_line_gap: 45
hough_accum_th: 100
goalpost_h_min: 203
goalpost_h_max: 232
goalpost_s_min: 54
goalpost_s_max: 255
goalpost_v_min: 65
goalpost_v_max: 206
field_filter: true
field_h_min: 203
field_h_max: 232
field_s_min: 54
field_s_max: 255
field_v_min: 65
field_v_max: 206
ellipse_size: 3
gain: 255
brightness: 105
exposure: 46
filter_debug: true